gateway/1.6/modules/ROOT/pages/index.adoc

```diff
@@ -49,6 +49,14 @@ A single Flex Gateway can support multiple backend APIs. To support more backend
 
 You can configure an optional external REDIS storage for features such as distributed rate limiting and distributed caching. Gateway replicas use the REDIS storage as a scratchpad to coordinate rate-limiting across multiple replicas, so customer SLAs are adhered to.
 
+== API Instances
+
+Flex Gateway supports HTTP and REST API instances.
+
+Flex Gateway does not natively support SOAP APIs and does not provide any schema validation for XML. However, you can publish an HTTP API instance to secure any API that uses HTTP protocol.
```

Comment:
Same comment as above:

"...does not provide any schema validation for XML."

At first glance I assumed this was talking about the schema validation policy. Is it actually referring to validating SOAP XML? If so, I wonder if there's a way to make it a little clearer? Or, I could be misunderstanding.